std_msgs/Header header 
int32 drone_id
UnassignedGrid[] unassigned_grids
int32[] local_search_seq